eng/pipelines/typespec-ci.yml

```
@@ -21,6 +21,6 @@ jobs:
     condition: succeededOrFailed()
 
   - pwsh: |
-      $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 $(Build.SourcesDirectory)
+      $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 $(Build.SourcesDirectory) -GitClean
```

```suggestion
      $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 $(Build.SourcesDirectory) --GitClean
```

It might work either way, but the convention for long option names is usually `--` instead of `-`

---

it works either way, but yeah I agree with following the convention